src/main/java/com/tileman/TilemanModeConfig.java

```diff
@@ -130,4 +126,16 @@ default boolean includeTotalLevel()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			keyName = "includeExp",
+			name = "Include Experience",
+			description = "Includes experience / 1000 in usable tiles",
+			section = customGameModeSection,
+			position = 4
+	)
+	default boolean includeExp()
```

Comment:
Can we change to "Exclude experience" and have it default to false. That way when you turn on custom game mode, it still behaves as expected without having to toggle this.

Reply:
Yeah that makes more sense 👍

Reply:
Fixed